lib/Minz/Extension.php

```diff
@@ -49,35 +40,36 @@ abstract class Minz_Extension {
 	 *
 	 * @param array{'name':string,'entrypoint':string,'path':string,'author'?:string,'description'?:string,'version'?:string,'type'?:'system'|'user'} $meta_info
 	 * contains information about the extension.
+	 * @throws Minz_ExtensionException
 	 */
 	final public function __construct(array $meta_info) {
 		$this->name = $meta_info['name'];
 		$this->entrypoint = $meta_info['entrypoint'];
 		$this->path = $meta_info['path'];
-		$this->author = isset($meta_info['author']) ? $meta_info['author'] : '';
-		$this->description = isset($meta_info['description']) ? $meta_info['description'] : '';
-		$this->version = isset($meta_info['version']) ? (string)$meta_info['version'] : '0.1';
-		$this->setType(isset($meta_info['type']) ? $meta_info['type'] : 'user');
+		$this->author = $meta_info['author'] ?? '';
+		$this->description = $meta_info['description'] ?? '';
+		$this->version = $meta_info['version'] ?? '0.1';
+		$this->setType($meta_info['type'] ?? 'user');
 
 		$this->is_enabled = false;
 	}
 
 	/**
 	 * Used when installing an extension (e.g. update the database scheme).
 	 *
-	 * @return string|true true if the extension has been installed or a string explaining the problem.
+	 * @return true true if the extension has been installed or a string explaining the problem.
 	 */
-	public function install() {
+	public function install(): bool {
 		return true;
 	}
```

Comment:
This is wrong. Please revert. This is supposed to be overridden by extensions inheriting this class